GDJS/Runtime/gd.ts

```diff
@@ -486,6 +486,33 @@ namespace gdjs {
       hex[r[15]]
     );
   };
+
+  const _console = {
+    info: console.log,
+    warning: console.warn,
+    error: console.error,
+  };
+
+  /**
+   * Internal method for logging messages to the JS console or the Debugger if available.
+   * Should be used in engine code or extensions, console.log is fine for JS events.
+   */
+  export let log = (
+    group: string,
+    message: string,
+    type: 'info' | 'warning' | 'error' = 'info'
+  ) => {
+    const logger = _console[type] || _console.info;
+    logger(`[${group}] ${message}`);
+  };
+
+  /**
+   * Shortcut loggin function for the runtime engine code.
```

Comment:
Typo in loggin

Reply:
But take a look at the "createLogger" function I was talking about instead :) This way each file can have its logger which is associated to a group and has a common syntax (otherwise we'll make more of these "shortcut" methods in the future, and it's not very elegant)